cmd/cloudFoundryDeleteService.go

```diff
@@ -16,6 +18,11 @@ func cloudFoundryDeleteService(CloudFoundryDeleteServiceOptions cloudFoundryDele
 
 	cloudFoundryLogin(CloudFoundryDeleteServiceOptions, &c)
 
+	if CloudFoundryDeleteServiceOptions.CfServiceKeys == true {
+		log.Entry().WithField("Delete Service Bindings and Keys flag is set", CloudFoundryDeleteServiceOptions.CfServiceKeys).Info("Deleting inherent Service Bindings and Service Keys")
```

Comment:
please don't use a sentence for a field.

I would suggest to either use a fitting noun or to not use the field and rather put the information into the message.

Reply:
Thanks for the suggestion. It's not using the field anymore, only Info now.